app/lib/html_css_to_image.rb

```diff
@@ -4,6 +4,8 @@ module HtmlCssToImage
 
   FALLBACK_IMAGE = "https://thepracticaldev.s3.amazonaws.com/i/g355ol6qsrg0j2mhngz9.png".freeze
 
+  CACHE_EXPIRATION = 2.years
```

Comment:
I don't think we ever want Redis to be so full that we have to use the eviction policy and I also think Redis data should stay pretty up to date. Anything that is stale we want to try and remove as soon as possible. With that said, what do you think about changing this to 1.month? cc @benhalpern

Reply:
I have very little context on y'alls infra, so whatever you think is best. 😄 

I think something longer than 1 month would be good, because we're often seeing images generated for old posts (1+ years old).

**Some more context on why I set it long:**
For caching in Rails, I've had success with never setting an expiration and letting the expiration policy do it's thing (this is what we do at GitHub). Then the metric we keep an eye on is cache hit ratio.

Reply:
> because we're often seeing images generated for old posts (1+ years old).

Do you have data for how often this happens? I would suspect, but totally could be wrong, that it does not happen very often and therefore hanging on to a lot of data and using up all the space in the cache probably isnt worth it on the off chance only a couple of keys are useful in the distant future.

Reply:
I don't have data without a lot of effort. I scanned through a few pages (100ish images), and only saw ~10 that are for articles from 2020. So right now it looks like the majority are older images.

Reply:
Hey, @mscoutermarsh I was just brought up to date on our usage and this account. Unfortunately, I don't have access to view it atm so I had no idea the majority of our images were that old. Ideally, I think the solution here is that these get stored with the objects they belong to in our Postgres db but since that is a much large project, what do you think about extending the cache to 6 weeks?

Reply:
@mstruve Updated!

If you all end up using Postgres. Doing a single table with a polymorphic association could save some time. Wouldn't need to add columns to all the tables that have images.

Only downside of that I see is losing the nice auto updating images when content in the HTML changes.

Reply:
HA you read our minds, that is exactly what we talked about!